src/core/meshcop/dataset_manager_ftd.cpp

```diff
@@ -372,13 +382,12 @@ Error ActiveDataset::GenerateLocal(void)
     {
         if (Get<KeyManager>().IsPskcSet())
         {
-            IgnoreError(dataset.SetTlv(Tlv::kPskc, Get<KeyManager>().GetPskc()));
+            IgnoreError(dataset.SetTlv(Tlv::kPskc, Get<KeyManager>().GetPskc().mLiteralKey));
         }
         else
         {
             // PSKc has not yet been configured, generate new PSKc at random
             Pskc pskc;
-
```

Comment:
Per our style guide, we need the new line after variable declarations. I noticed the same in other places in the code.

> - Scoped Variable declarations
>   - All scoped (i.e. stack) variable declarations should be placed together at the top of the enclosing scope in which they are used.
>   - There shall be an empty line after all such variable declarations.
>   - The names of all variable declarations should be left aligned.
>

Reply:
Sorry I missed that, will update the PR. But shouldn't make pretty take care of this?

Reply:
Unfortunately, I don't think there is any way to add this with `clang-format`. I agree it would be much easier if the tools could help with this (or maybe even just check it). It does take care of a lot of other style related things though.